### PR TITLE
Add Tezos support

### PR DESCRIPTION
**Pull request by aliel**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -64,6 +64,7 @@ testing =
     base58
     fastapi
     requests
+    aleph_pytezos
```
Comment (hoh) [inline setup.cfg:67]:
```suggestion
    aleph_pytezos@git+https://github.com/aleph-im/aleph-pytezos.git@faebbaa99f4cc8a16303efa4ad86899dd0443e5b
```

**Review by hoh — COMMENTED** [review]
Looks good overall, except I would recommend the install of aleph-pytezos from git instead of PyPI at the moment.

File: tests/unit/test_chain_tezos.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    account: TezosAccount = get_fallback_account()
+
+    message = Message("TEZOS", account.get_address(), "SomeType", "ItemHash")
+    signed = await account.sign_message(asdict(message))
+    assert signed["signature"]
+    assert len(signed["signature"]) == 99
+
+    address = account.get_address()
+    assert address
+    assert type(address) == str
```
Comment (odesenfans) [inline tests/unit/test_chain_tezos.py:34]: `assert isinstance(address, str)` is more pythonic I'd say.

File: tests/unit/test_chain_tezos.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+async def test_TezosAccount():
+    account: TezosAccount = get_fallback_account()
+
+    message = Message("TEZOS", account.get_address(), "SomeType", "ItemHash")
+    signed = await account.sign_message(asdict(message))
+    assert signed["signature"]
+    assert len(signed["signature"]) == 99
+
+    address = account.get_address()
+    assert address
```
Comment (odesenfans) [inline tests/unit/test_chain_tezos.py:33]: If you're checking against `None`, `assert address is not None` is clearer.

File: tests/unit/test_chain_tezos.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+def test_get_fallback_account():
+    delete_private_key_file()
+    account: TezosAccount = get_fallback_account()
+
+    assert account.CHAIN == "TEZOS"
+    assert account.CURVE == "secp256k1"
+    assert account._account.public_key()
+
+@pytest.mark.asyncio
+async def test_TezosAccount():
```
Comment (odesenfans) [inline tests/unit/test_chain_tezos.py:24]: We use snake case for function names, so `test_tezos_account` is more idiomatic.

File: src/aleph_client/chains/tezos.py [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        return message
+
+    def get_address(self) -> str:
+        return self._account.public_key_hash()
+
+    def get_public_key(self) -> str:
+        return self._account.public_key()
+
+    async def encrypt(self, content) -> bytes:
+        value: bytes = bytes(SealedBox(self._private_key.public_key).encrypt(content))
```
Comment (odesenfans) [inline src/aleph_client/chains/tezos.py:42]: The type annotation seems superfluous, and I wonder if you need to cast to bytes as well. You don't do it for the decrypt function?

**Review by odesenfans — COMMENTED** [review]
Minor comments.

**Pull request merged**